plugins/influxdb/influx.go

```diff
@@ -0,0 +1,26 @@
+package influxdb
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func InfluxDBCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "InfluxDB CLI",
+		Runs:    []string{"influx"},
+		DocsURL: sdk.URL("https://docs.influxdata.com/influxdb/"),
```

Comment:
Would this be a better link?

- https://docs.influxdata.com/influxdb/cloud/tools/influx-cli/

Would you also know if the InfluxDB CLI supports both InfluxDB Serverless and InfluxDB Cloud 2.0?

Reply:
Thanks, updated it.